docs/developer/validation-rules/x-ms-enum.md

```diff
@@ -0,0 +1,5 @@
+# ValidFormats
+## Description
+For enum types, we should encourage authors to use the `x-ms-enum` extension and apply appropriate options. Please refer to the dcoumentation details [here](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md#x-ms-enum)
+## How to fix
+Add the `x-ms-enum` extension to the enum and set appropriate properties including `modelAsString`.
```

Comment:
should we write here what you pointed out before... setting modelAsString: true, can help avoid SDK breaking changes in the future if you think there could be more values added to the enum. ?